modules/monitoring-managing-user-permissions-using-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="managing-user-permissions-using-cli_{context}"]
+= Managing user permissions using CLI
+
+This procedure shows how to manage user permissions for monitoring their own services using the CLI.
+
+.Prerequisites
+
+* You need to have a user created.
+* Log in using the `oc` command.
+
+.Procedure
+
+This example shows how to assign permissions to user `johnsmith` for setting up metrics collection and creating alerting rules in the `ns1` namespace:
+
+* Run this command to assign <role> to <user> in <namespace>:
++
+----
+$ oc policy add-role-to-user <role> <user> -n <namespace>
+----
++
+Substitute <role> for `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit`.
```

Comment:
Is this a separate step? I don't think so, and if not, I would either incorporate it before the code block or add it as an annotation so that a user knows by the time they enter the command.

Reply:
I think this is fine, and is used throughout out documentation. Even in the unlikely scenario that they run the command without substituting anything, nothing will happen. The problem with moving this to before the code block is that it would get separated from the explanation of the three roles. How exactly do you see using an annotation here?

Reply:
That makes sense, Max. Looking at it with edits and fresh eyes and I think it works. I agree that moving it before the code block is not a great idea. 

To answer your question, I was thinking of adding an annotation at the end of the command:

> $ oc policy add-role-to-user <role> <user> -n <namespace> **<1>**
----
+
**<1>** Substitute <role> for `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit`.

That way, the user would be able to avoid typing the command, hitting enter, and then reading the instruction to substitute those parameters. But I do see one of the problems with this solution is that the annotation note would be very long because you'd need to include all 3 roles. So disregard. :)